perception/autoware_elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -66,22 +66,22 @@ ElevationMapLoaderNode::ElevationMapLoaderNode(const rclcpp::NodeOptions & optio
   param_file_path_ = this->declare_parameter("param_file_path", "path_default");
   map_frame_ = this->declare_parameter("map_frame", "map");
   bool use_sequential_load = this->declare_parameter<bool>("use_sequential_load", true);
-  int sequential_map_load_num_int = this->declare_parameter<int>("sequential_map_load_num", 1);
+  int sequential_map_load_num_int = this->declare_parameter<int>("sequential_map_load_num");
   if (sequential_map_load_num_int > 0) {
     sequential_map_load_num_ = (unsigned int)sequential_map_load_num_int;
   } else {
     throw std::runtime_error("sequential_map_load_num should be larger than 0.");
   }
   use_inpaint_ = this->declare_parameter("use_inpaint", true);
```

Comment:
We should remove default value from this as well. There seems to be other parameters that need to remove default parameters. @vish0012 could you update this branch and fix other parameters as well?